server/models/DB.sql

```diff
@@ -8,26 +8,36 @@ CREATE TABLE users (
   username varchar (100) NOT NULL UNIQUE,
   email varchar (50) NOT NULL UNIQUE,
   password varchar (100) NOT NULL,
-  is_landlord boolean DEFAULT false,
-  landlord_id integer,
-  created_at timestamp DEFAULT CURRENT_TIMESTAMP
+  profile_pic VARCHAR DEFAULT 'userProfile.png',	
+  is_landlord boolean DEFAULT false NOT NULL,
 );
 
 CREATE TABLE landlords(
   _id SERIAL PRIMARY KEY,
-  first_name varchar (50) NOT NULL,
-  last_name varchar (50) NOT NULL,
-  full_name varchar (100) NOT NULL,
-  profile_pic VARCHAR DEFAULT 'userProfile.png',	
   overall_rating decimal,
   respect_rating decimal,
   responsiveness_rating decimal,
   bike_friendly boolean,
   pet_friendly boolean,
   is_verified boolean DEFAULT false,
-  created_at timestamp DEFAULT CURRENT_TIMESTAMP
+  FOREIGN KEY (_id) REFERENCES users(_id),
```

Comment:
This should be `FOREIGN KEY (user_id) REFERENCE users(_id)`. Each table should have it's own unique primary key (`_id`), so you'll want to add an additional column to the table specifically for the foreign key.